drivers/pds/lib/utils.go

```diff
@@ -1487,12 +1551,32 @@ func CreateDataServiceWorkloads(dataServiceName string, deploymentID string, sca
 
 	case cassandra:
 		cassCommand := deploymentName + " write no-warmup n=1000000 cl=ONE -mode user=pds password=" + pdsPassword + " native cql3 -col n=FIXED\\(5\\) size=FIXED\\(64\\)  -pop seq=1..1000000 -node " + dnsEndpoint + " -port native=9042 -rate auto -log file=/tmp/" + deploymentName + ".load.data -schema \"replication(factor=3)\" -errors ignore; cat /tmp/" + deploymentName + ".load.data"
-		dep, err = CreatecassandraWorkload(cassCommand, deploymentName, namespace)
+		dep, err = CreateDeploymentWorkloads(cassCommand, deploymentName, cassandraStresImage, namespace)
 		if err != nil {
 			log.Errorf("An Error Occured while creating cassandra workload %v", err)
 			return nil, nil, err
 		}
+	case elasticSearch:
+		esCommand := "while true; do esrally race --track=geonames --target-hosts=" + dnsEndpoint + " --pipeline=benchmark-only --test-mode --kill-running-processes --client-options=\"timeout:60,use_ssl:false,verify_certs:false,basic_auth_user:'elastic',basic_auth_password:" + pdsPassword + "\"; done"
```

Comment:
Let's have the options to this command configurable - for tuning the workload better in future tests like scale, longevity.

Reply:
Done